agent/bench-scripts/pbench-fio

```diff
@@ -641,8 +604,6 @@ function fio_run_benchmark() {
 			for block_size in `echo $block_sizes | sed -e s/,/" "/g`; do
 				job_num=1
 				iteration="${count}-${test_type}-${block_size}KiB"
-				   
-                                echo $iteration >> $benchmark_iterations
```

Comment:
Yes, but the record_iteration function is not in master yet (only in wip-indexing). I'l fix this up after I PR the iterations commit.